rules/rules.libsonnet

```diff
@@ -358,6 +358,34 @@
               )
             ||| % $._config,
           },
+          {
+            record: 'node:disk_inodes_total:',
+            expr: |||
+              max(
+                max(
+                  kube_pod_info{%(kubeStateMetricsSelector)s, host_ip!=""}
+                ) by (node, host_ip)
+                * on (host_ip) group_right (node)
+                label_replace(
+                  (max(node_filesystem_files{mountpoint="/"}) by (instance)), "host_ip", "$1", "instance", "(.*):.*"
```

Comment:
This label would be `mountpoint="/host/root"` for me. I guess we would need to make it configurable. Additionally we should also put the `nodeExporterSelector` to these matching labels.

Reply:
Got it, I'll figure out how to make it configurable

Reply:
Yelp, these rules could be used for a dashboard.
Will make the changes as well. 

Thanks for the suggestion,
Ben